src/log-store/src/kafka/consumer.rs

```diff
@@ -60,40 +61,61 @@ struct FetchResult {
     used_offset: i64,
 }
 
+const MAX_BATCH_SIZE: usize = 52428800;
+const AVG_RECORD_SIZE: usize = 256 * 1024;
```

Comment:
Can they be `ReadableSize`?

Reply:
Resolved in [35280f7](https://github.com/GreptimeTeam/greptimedb/pull/4510/commits/35280f773a591ea6d8a74d570c8722d02c930f6e)